chapters/cls-metrics.qmd

```diff
@@ -0,0 +1,1693 @@
+---
+knitr:
+  opts_chunk:
+    cache.path: "../_cache/cls-metrics/"
+---
+
+# Characterizing Classification Models {#sec-cls-metrics}
+
+```{r}
+#| label: cls-metrics-setup
+#| include: false
+
+source("../R/_common.R")
+source("../R/_themes.R")
+source("../R/_themes_ggplot.R")
+
+# ------------------------------------------------------------------------------
+
+library(tidymodels)
+library(discrim)
+library(probably)
+library(desirability2)
+library(patchwork)
+library(ggrepel)
+library(kableExtra)
+library(cli)
+library(gt)
+
+# ------------------------------------------------------------------------------
+# set options
+
+tidymodels_prefer()
+theme_set(thm_lt)
+
+set_options()
+
+# ------------------------------------------------------------------------------
+# load data 
+
+load("../RData/wordle_results.RData")
+load("../RData/one_user_data_splits.RData")
+load("../RData/mushrooms.RData")
+```
+
+```{r}
+#| label: mushrooms-nb
+#| include: false
+#| warnnig: false
+#| cache: true
+
+shroom_n <- nrow(shroom_train) +
+  nrow(shroom_val) +
+  nrow(shroom_cal) +
+  nrow(shroom_test)
+shroom_xtab <- round(table(shroom_train$class) / nrow(shroom_train) * 100, 1)
+obs_prev <- round(mean(shroom_train$class == "poisonous") * 100, 1)
+
+cls_mtr <-
```

Comment:
I used [air](https://www.tidyverse.org/blog/2025/02/air/) to format the R. It does pretty well but these lines are _a choice_.